pkg/services/authapi/fake/authapistub.go

```diff
@@ -0,0 +1,69 @@
+package fake
+
+import (
+	"context"
+	"fmt"
+	"github.com/grafana/grafana/pkg/services/authapi"
+
+	"github.com/grafana/grafana/pkg/util"
```

Comment:
The linter is complaining about this I think:
```suggestion

	"github.com/grafana/grafana/pkg/services/authapi"
	"github.com/grafana/grafana/pkg/util"
```